src/table/utils/color-utils.js

```diff
@@ -34,25 +43,33 @@ export function resolveExpression(input) {
 }
 
 export function isDarkColor(color) {
-  let r;
-  let g;
-  let b;
-  let matches;
-  if (
-    (matches =
-      /^rgb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/i.exec(color) ||
-      /^rgba\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d(\.\d+)?)\s*\)$/i.exec(color))
-  ) {
-    r = parseInt(matches[1], 10);
-    g = parseInt(matches[2], 10);
-    b = parseInt(matches[3], 10);
-  } else if ((matches = /^#([A-f0-9]{2})([A-f0-9]{2})([A-f0-9]{2})$/i.exec(color))) {
```

Comment:
what happened to this case?

Reply:
const rgba = resolveExpression(color);
color was converted first, so no need to handle the hex code

Reply:
The thing is that this file exists in multiple repos. Optimally they should be moved to a separate package. Before that is done, we want to keep it as similar as possible, to easily extract it. If I understand what you are trying to do, you need the color on a rgba format for `isTransparentColor` and `removeOpacity`. The thing is that hex will never have transparancy so you can early return in that case

Reply:
the hex has transparency when there are eight digits https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4
So in https://github.com/qlik-oss/sn-table/pull/517/files#diff-116e1caa786a873fe51a85a99bf012cc7b016e631b783c26e99cac13ee0eb1e5R9, we should get the alpha value from the hex code.

Reply:
Ahh that is new to me. ok so what resolve expression should do is return a value that is valid css. Hmm this is tricky. So if you really want to do it this way, I think you should rewrite it to a `resolveToRGBA` function. Or just add a case to `isTransparentColor` and `removeOpacity` that covers hexcolors with eight digits

Reply:
okay, I will rename the resolveExpression to resolveToRGBAorRGB

Reply:
Done